src/main/plugins/org.dita.pdf2/cfg/fo/i18n/ko.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- 
+Copyright © 2004-2006 by Idiom Technologies, Inc. All rights reserved. 
```

Comment:
Good point. How do you want me to handle the commits at that point. Earlier ones were mostly separated by what they were doing - add variables, add i18n, add individual index. I'm fine with squashing those all into one commit, along with this update, FWIW.